apps/intercom/src/connectors/users.ts

```diff
@@ -0,0 +1,74 @@
+import { z } from 'zod';
+import { IntercomError } from './commons/error';
+
+const intercomUserSchema = z.object({
+  id: z.string(),
+  name: z.string(),
+  email: z.string().optional(),
+});
+
+export type IntercomUser = z.infer<typeof intercomUserSchema>;
+
+const intercomResponseSchema = z.object({
+  pages: z.object({
+    page: z.number().nullable(),
+    per_page: z.number().nullable(),
+    next: z.object({
+      starting_after: z.string().nullable(),
+    }).optional(),
+  }).optional(),
+  admins: z.array(intercomUserSchema),
```

Comment:
```suggestion
  admins: z.array(z.unknown()),
```
The point is to let unexpected admin data pass the `.parse(data)` validation, then separate the invalid admins from the the valid one.